packages/documentation/pages/usage/components/filters.vue

```diff
@@ -8,6 +8,7 @@
     			<KtFilters
     				v-model="filters"
     				:columns="componentProps.columns"
+    				dataTest="kt-filters"
```

Comment:
Won’t work with more than 1 instance of `KtFilters`

Reply:
This is the usage example of `KtFilters`. Isn't the user supposed to make sure to provide unique strings for `dataTest` when using multiples instances of KtFilters?